src/pages/explore/wallets/index.page.tsx

```diff
@@ -0,0 +1,57 @@
+import { Container } from "@components/commons/Container";
+import { SSRConfig } from "next-i18next";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { ExploreCards, ExploreCardsImage } from "../_components/ExploreCards";
+
+export default function ExploreWallets() {
+  return (
+    <Container className="mb-10">
+      <ExploreCards
+        title="EXPLORE DEFICHAIN"
+        desc="Get $DFI"
+        bgImage={ExploreCardsImage.METALCOIN}
+        href=""
+      />
+      <ExploreCards
+        title="EXPLORE DEFICHAIN"
+        desc="Trade with DEX"
+        bgImage={ExploreCardsImage.DIAMONDCOIN}
+        href=""
+      />
+      <ExploreCards
+        title="EXPLORE DEFICHAIN"
+        desc="Running Masternodes"
+        bgImage={ExploreCardsImage.CUBE}
+        href=""
+      />
+      <div className="flex flex-row overflow-x-scroll gap-x-5">
+        <ExploreCards
+          title="EXPLORE DEFICHAIN"
+          desc="Choose your wallet"
+          bgImage={ExploreCardsImage.PHONE}
+          href=""
+        />
+        <ExploreCards
+          title="EXPLORE DEFICHAIN"
+          desc="Invest, Trade & Earn"
+          bgImage={ExploreCardsImage.ARROW}
+          href=""
+        />
+      </div>
+    </Container>
+  );
+}
+
+export async function getStaticProps({
+  locale,
+}): Promise<{ props: SSRConfig }> {
+  return {
+    props: {
```

Comment:
are we using the props return here? 
i dont see it on line 6

Reply:
its for the translations on the navbar if i dont return translations wouldnt take effect